src/util/path.js

```diff
@@ -507,7 +507,9 @@
         p, nextLen, nextStep = 0.01, angleFinder = segInfo.angleFinder, lastPerc;
     // nextStep > 0.0001 covers 0.00015625 that 1/64th of 1/100
     // the path
-    while (tmpLen < distance && perc <= 1 && nextStep > 0.0001) {
+    
+    // floating point made perc <= 1 ignore some calculation, so perc < 1.1 do the trick
+    while (tmpLen < distance && perc < 1.1 && nextStep > 0.0001) {
```

Comment:
why did they even bother adding floats to JS.

Reply:
You left some trailing spaces, lint is failing.
This is good to go probably.
Can we look into reducing 1.1 to minimum possible? 1.01? 1.1 would be 10% of tolerance seems high.
I would also like to see a test that fails with 1 and pass with 1.0x

Reply:
I think the correct should be
```
while (tmplen < distance && perc < (perc + nextstep) && nextstep > 0.0001)
```